### ISSUE TITLE
Number of segments seems incorrect?

### ISSUE THREAD

**Issue by eugenevinitsky** [waymo-research/waymo-open-dataset#436]
The datasheet lists the number of segments as about 103000. After processing the tf records I have 487004 segments; I understand the data is split into 9 second chunks with 5 seconds of overlap but I can't seemingly find any logic by which I could get to 487004 total 9 second segments. Is it possible the motion v 1.1 dataset has a few additional segments?

Thanks!

**Comment by scott-ettinger** [comment]
Here is some context. The dataset splits the 103000 20 second segments into a training set (~68% or 69572 segments), a validation set (~16%) and a test set (~16%).  9 second windows are then generated from these 20 second segments.  From each training set scenario, 7 overlapping 9 second windows are generated. The 5 second overlap statement is not strictly correct, the exact splits can be found in the paper here : https://arxiv.org/abs/2104.10133. Thanks for pointing this out, we will update this on the website.  To arrive at the number of examples in the training set, multiply the training set 69572 segments by 7 and you arrive at the 487004 examples.

**Comment by eugenevinitsky** [comment]
Thank you so much! That clarifies things a lot.
